lib/overcommit/utils.rb

```diff
@@ -211,6 +211,45 @@ def execute_in_background(args)
         Subprocess.spawn_detached(args)
       end
 
+      # Return the number of processors used by the OS for process scheduling.
+      #
+      # @see https://github.com/grosser/parallel/blob/v1.6.1/lib/parallel/processor_count.rb#L17-L51
+      def processor_count # rubocop:disable all
+        @processor_count ||=
+          begin
+            os_name = RbConfig::CONFIG['target_os']
+            if os_name =~ /mingw|mswin/
```

Comment:
Yes, I had simply copied this wholesale from the `parallel` gem (linked above). 

While `os_name` is used further down, the number of systems using Overcommit that would match that `elsif` branch is likely zero for the foreseeable future, so I'll remove it. If it's an issue, someone can make a separate pull request. :)